src/libs/actions/IOU.ts

```diff
@@ -6719,6 +6719,29 @@ function getPayMoneyRequestParams(
         });
     }
 
+    if (full) {
```

Comment:
Can you add a comment here about why we modify the onxyData if `full` is `true`

Reply:
Done